feat!: Support tenant creation and consolidate api

flaki opened the pull request:
Add missing SDK features to be able to support a simple SE2 integration.

callahad commented on README.md:
```diff
@@ -1,12 +1,12 @@
 <p align="center">
     <a href="https://suborbital.dev/">
-        <img src="suborbital-logo.png" alt="Suborbital" height="200" />
+        <img src="suborbital-logo.svg" alt="Suborbital" height="120" />
```
Should we also `git rm` the old png?

callahad replied:
(Though logo stuff is probably best as a separate PR, semantically...)

flaki commented:
I have added the required Tenant APIs to get the demo working with the upstream version once this gets published.

I have also moved the session token API from the builder to the admin which is a better place for it given that it uses the environment token, just like the Tenant API.

Currently the hosted API URLs point to staging but I made it configurable through `apiUri` in the config.

flaki commented on src/admin.ts:
```diff
@@ -35,28 +36,74 @@ interface ExecutionResult {
 const ADMIN_URI =
   "http://se2-controlplane-service.suborbital.svc.cluster.local:8081";
 
+const API_URI = "https://api.stg.suborbital.network";
```
I removed the custom bits and pointed the fallback address to staging.

If we wanted to keep backwards compat we could probably keep the controlplane URI as a default (i'm setting it manually from the demo anyway), but it may not matter given the changes abound.
